src/Api/Model/Languageforge/Lexicon/Command/LexUploadCommands.php

```diff
@@ -125,11 +126,16 @@ public static function uploadAudioFile($projectId, $mediaType, $tmpFilePath)
 
             }
 
+            //Find the size of the upload file now pointed to by the entry
+            $fileSize = intval(`ffprobe -i $filePath -show_entries format=size -v quiet -of csv="p=0"`);
```

Comment:
While this works, using ffprobe to get the file size of a file is probably overkill.  How about this?
```suggestion
            $fileSize = intval(`ls -l $filePath | awk '{print $5}'`);
```

Reply:
In the latest commit I use the php filesize command for both audio and images.

Reply:
thank you